zipkin-collector/scribe/pom.xml

```diff
@@ -50,6 +50,7 @@
     <dependency>
       <groupId>javax.annotation</groupId>
       <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.1</version>
```

Comment:
Sorry, a bit late with my comment, wouldn't it be better to define `javax.annotation-api` version in the parent `pom.xml`? I see it being used in a few places.

Reply:
sure. want to raise a PR to add that property? and use it in the few?